test/RSocketTests.h

```diff
@@ -8,6 +8,11 @@
 
 #include "rsocket/transports/tcp/TcpConnectionFactory.h"
 
+auto const default_baton_timeout = std::chrono::milliseconds(100);
+#define CHECK_WAIT(baton) CHECK(baton.timed_wait(default_baton_timeout))
+// #define CHECK_WAIT(baton) baton.timed_wait(default_baton_timeout)
```

Comment:
I get that they're commented out so they can be easily swapped in, but I don't think it's necessary and clutters things up.